pycloudlib/oci/cloud.py

```diff
@@ -37,6 +38,19 @@ def __init__(
             config_path: Path of OCI config file
         """
         super().__init__(tag, timestamp_suffix)
+        if not compartment_id:
+            command = ['oci', 'iam', 'compartment', 'get']
+            exception = Exception(
+                "Could not obtain OCI compartment id. Has the CLI client been "
+                "setup?\nCommand attempted: '{}'".format(' '.join(command))
+            )
+            try:
+                result = subp(command)
+            except FileNotFoundError as e:
+                raise exception from e
+            if not result.ok:
+                raise exception
```

Comment:
This is nit-level, so don't worry too much about it: will this include the output from the failing command? (That would save folks from having to re-run the command to see what the exact error was.)